index.html

```diff
@@ -146,5 +146,31 @@ <h2>Skills</h2>
             </li>
         </ul>
     </section>
+    <section class="contact">
+        <footer class="contact-form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xlevdjlz" method="POST">
+            <ul>
+                <li>
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" placeholder="Your Name" maxlength="30">
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="Enter your email">
+                </li>
+                <li>
+                    <label for="message"></label>
+                    <input type="text" name="user_message" id="message" placeholder="write your message here" maxlength="500">
+                </li>
```

Comment:
- [ ] Please add a `required` attribute to all the input fields. This is so that the user cannot submit the form if one or more o the fields are empty as demanded by the project requirements.

- [ ] Kindly use a `<textarea>` element for the message field. This is best for semantics.